src/esp/agent/Agent.h

```diff
@@ -60,7 +61,7 @@ struct AgentConfiguration {
   float coefficientOfRestitution = 0.0;
 
   std::vector<sensor::SensorSpec::ptr> sensorSpecifications = {
-      sensor::SensorSpec::create()  // default SensorSpec
+      sensor::CameraSensorSpec::create()  // default CameraSensorSpec
```

Comment:
@Skylion007 : By default, there is a "default" sensor spec in the list of sensor specifications.
Are we using it directly in the python?

I do not think there is any reason to have such a "default" sensor spec taking up the entry [0] in the vector. This list should be empty at the very beginning. User push sensor specifications to this list from scratch.

Reply:
Nah, we don't use this interface I don't think.

Reply:
Thanks, I changed this so that the default AgentConfiguration has sensorSpecifications empty, and the demo in viewer has the user add a sensorSpec, as we do in SimTest.